### PR TITLE
feat: ab test onboarding title and subtitle

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- AB test the onboarding's both title and subtitle.
- As discussed during the meeting, the feature value should be what is expected as opposed to the usual values of v1, v2, etc.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-2018 #done

### CODE REVIEW

File: packages/webapp/pages/onboarding.tsx [hunk 1]
```diff
@@ -347,11 +353,11 @@ export function OnboardPage(): ReactElement {
             )}
           >
             <OnboardingTitleGradient className="mb-4 typo-large-title tablet:typo-mega1">
-              Where developers grow together
+              Where developers {onboardingTitle} together
```
Comment (rebelchris) [inline packages/webapp/pages/onboarding.tsx:356]: I suggest we put the entire title in the string not just the word.

**Review by rebelchris — CHANGES_REQUESTED** [review]
1 small request 🙏

**Review by rebelchris — APPROVED** [review]
Looks good to me.
@idoshamun anything from your side?

**Comment by idoshamun** [comment]
@rebelchris can we make it one feature that returns both values? Otherwise experimentation will be much harder to set up.

**Comment by rebelchris** [comment]
Ah good point forgot to add the check.
@sshanzel could you check if it supports json object with 2 strings for instance?

**Review by capJavert — APPROVED** [review]
Looks good from my end. Would love to see if json stuff works 👍

**Comment by sshanzel** [comment]
Have applied the object structure for this feature with the key of `onboarding_copy`.

Screenshot 2023-12-05 at 6 50 53 PM

**Review by idoshamun — APPROVED** [review]

**Review by rebelchris — APPROVED** [review]
Amazing amazing

**Pull request merged**